src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -91,20 +91,25 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
   const { user, profile } = useUser();
   const venueId = useVenueId();
   const venue = useSelector(currentVenueSelectorData);
+  const venueParentId = venue?.parentId;
   const privateChats = useSelector(privateChatsSelector);
   const radioStations = useSelector(radioStationsSelector);
   const parentVenue = useSelector(parentVenueSelector);
 
-  const venueParentId = venue?.parentId;
-  const venueParentQuery = useMemo<ReduxFirestoreQuerySetting>(
-    () => ({
-      collection: "venues",
-      doc: venueParentId,
-      storeAs: "parentVenue",
-    }),
-    [venueParentId]
+  const venueParentQuery = useMemo<SparkleRFQConfig[]>(
+    () =>
+      venue?.parentId
+        ? [
+            {
+              collection: "venues",
+              doc: venue.parentId,
+              storeAs: "parentVenue",
+            },
+          ]
+        : [],
+    [venue]
   );
-  useFirestoreConnect(venueParentId ? venueParentQuery : undefined);
+  useSparkleFirestoreConnect(venueParentId ? venueParentQuery : undefined);
```

Comment:
As above, RE: recombining the query here

```suggestion
  useSparkleFirestoreConnect(venueParentId ? {
      collection: "venues",
      doc: venueParentId,
      storeAs: "parentVenue",
    } : undefined);
```